### PR TITLE
[DOCS] Corrected typo 'virutal' corrected to 'virtual'

### PR DESCRIPTION
**Pull request by dewmeht**
Corrected typo 'virutal' corrected to 'virtual'

Changes proposed in this pull request:
- Correction of typo in one word

### Definition of Done
Please delete options that are not relevant.

- [ ] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/en/latest/contributing/style_guide.html?highlight=style%20guide)
- [ ] I have performed a [self-review](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html?highlight=checklist) of my own code

Thank you for submitting!

### CODE REVIEW

File: docs/guides/setup/installation/local.md [hunk 1]
```diff
@@ -121,4 +121,4 @@ You can confirm that great_expectations was successfully installed with:
 great_expectations --version
 ```
 
-<Congratulations />
\ No newline at end of file
+<Congratulations />
```
Comment (kenwade4) [inline docs/guides/setup/installation/local.md:124]: What specifically was changed here on the last line?

**Review by kenwade4 — COMMENTED** [review]
Hi @dewmeht thanks for fixing the typo. Once you get the CLA signed, we should be close to approval. See https://docs.greatexpectations.io/docs/contributing/contributing_misc/

**Comment by dewmeht** [comment]
Hello @kenwade4,
Can you help me understand what is a CLA and how do I get it signed.Sorry, but this is my first ever Open Source contribution.
Nothing was changed on the last line.
In docs/guides/setup/installation/local.md:
> \ No newline at end of file
+<Congratulations />

***@***.***
    On Wednesday, February 2, 2022, 12:24:55 PM EST, kenwade4 ***@***.***> wrote:  
 
 


@kenwade4 commented on this pull request.

Hi @dewmeht thanks for fixing the typo. Once you get the CLA signed, we should be close to approval.

In docs/guides/setup/installation/local.md:
> \ No newline at end of file
+<Congratulations />

What specifically was changed here on the last line?

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by kenwade4** [comment]
@dewmeht The link I included shows how to sign the CLA (Contributor License Agreement) https://docs.greatexpectations.io/docs/contributing/contributing_misc/

**Comment by kenwade4** [comment]
@dewmeht Once you sign the CLA (Contributor License Agreement), feel free to re-open this pull request.

**Pull request closed without merging**